README.md

```diff
@@ -1,59 +1,47 @@
-# Getting Started with Create React App
+# clockwork
+[![Contributors][contributors-shield]][contributors-link]
+[![Stargazers][stars-shield]][stars-link]
+[![Issues][issues-shield]][issues-link]
+[![Website][website-shield]][website-link]
 
-This project was bootstrapped with [Create React App](https://github.com/facebook/create-react-app).
+clockwork is a website built to simplify the experience of searching courses and building timetables for the [University of Victoria](https://uvic.ca) (UVic). clockwork is a TypeScript app that uses React on the front-end and Firebase on the back-end.
 
-## Available Scripts
+## Contribute
+1. Clone the repo:
+    ```
+    git clone https://github.com/VikeLabs/clockwork.git
+    ```
+2. Run `npm install`
+4. Find an unassigned task on [ZenHub](https://app.zenhub.com/workspaces/team-schedule-courses-5f973f50ae36d70012eb5b2e/board?repos=216653028) to work on.
+    * Filter repos to show issues for `clockwork`
+5. Run `npm start` to view any front-end changes on `localhost:3000`
+6. Create a new branch using `git checkout -b <branch-name>` (make sure it's up to date with `main`)
+7. Commit the changes you've made and push to GitHub to create a Pull Request.
 
-In the project directory, you can run:
+## Testing
 
-### `npm install`
+This project uses [Jest](https://jestjs.io/) testing framework. You can execute tests by running `npm test`.
 
-**Note:** Downgrade to NPM V6 for this repo to work correctly. Storybook 6.1.18 is having issues with NPM V7.\
-Use `npm install -g npm@6` to downgrade to NPM V6\
-https://github.com/storybookjs/storybook/issues/12983
+This will execute tests using Jest files with the extension `*.test*`.
 
-### `npm start`
+`npx jest --watch` will put Jest into watch mode, which will execute tests as files change.
 
-Runs the app in the development mode.\
-Open [http://localhost:3000](http://localhost:3000) to view it in the browser.
+## Developer Tools
 
-The page will reload if you make edits.\
-You will also see any lint errors in the console.
+When developing in your local environment, to view components which require requests to populate with data you will need to enable CORS. Install the following extension to do so:
```

Comment:
Might be nice to add something about Safari:
https://stackoverflow.com/questions/4556429/disabling-same-origin-policy-in-safari
This is what I did